Remove metrics object from sensctl event response (#1400)
sensu/sensu-docs: content/sensu-go/5.5/getting-started/learn-sensu.md

{{< highlight shell >}}
sensuctl event info sensu-go-sandbox curl_timings --format json | jq .
...
  "history": [
    {
      "status": 0,
      "executed": 1556472457
    },
  ],

>   "output": "sensu-go-sandbox.curl_timings.time_total 0.005 1556472657\n...

Reviewer: ```suggestion
  "output": "sensu-go-sandbox.curl_timings.time_total 0.005 1556472657\n...",
```

  ...
  "output_metric_format": "graphite_plaintext",
  "output_metric_handlers": [
    "influx-db"
  ],
...
{{< /highlight >}}

> Because we configured a metric format, the Sensu agent is able to convert the Graphite-formatted metrics provided by the check command into a set of Sensu-formatted metrics (not shown int he output), which are then sent to the InfluxDB handler that knows how to read Sensu-formatted metrics and convert them to a format InfluxDB will accept.

Reviewer: ```suggestion
Because we configured a metric format, the Sensu agent is able to convert the Graphite-formatted metrics provided by the check command into a set of Sensu-formatted metrics (not shown in the output), which are then sent to the InfluxDB handler that reads Sensu-formatted metrics and converts them to a format InfluxDB accepts.
```
